client/command.go

```diff
@@ -104,7 +109,24 @@ func Run(
 				cmd.Flag.Parse(args[1:])
 				args = cmd.Flag.Args()
 			}
-			cmd.Run(cmd, args)
+			errorStatus := cmd.Run(cmd, args)
+			var metricsKey string
+			if errorStatus != nil {
+				if errorStatus.serverError {
+					metricsKey = "failure"
+				} else {
+					metricsKey = "ignored_failure"
+				}
+			} else {
+				metricsKey = "success"
+			}
+			clientInvokeMetrics(map[string]string{
+				"metrics_key": metricsKey,
+				"method_name": cmd.Name(),
```

Comment:
you may want to prefix the method_name here with "client" so that it couldn't collide with names that are the same on the server.